package.json

```diff
@@ -3,9 +3,8 @@
   "version": "1.0.0",
   "description": "",
   "main": "index.js",
-  "type": "module",
   "scripts": {
-    "test": "jest --collectCoverage",
+    "test": "jest --collectCoverage --forceExit",
```

Comment:
That is not a good solution. If you have something that doesn't let your tester exit gracefully, find it and fix it